networkx/algorithms/graph_hashing.py

```diff
@@ -179,13 +184,13 @@ def weisfeiler_lehman_subgraph_hashes(
     along the connecting edge from this neighbor to node $n$. The resulting string
     is then hashed to compress this information into a fixed digest size.
 
-    Thus, at the $i$th iteration nodes within $2i$ distance influence any given
+    Thus, at the $i$-th iteration, nodes within $i$ hops influence any given
     hashed node label. We can therefore say that at depth $i$ for node $n$
     we have a hash for a subgraph induced by the $2i$-hop neighborhood of $n$.
```

Comment:
I'm confused as to why this $2i$ was changed to $i$ but the $2i$ two lines down was left as $i$.

1) should they be different?
2) where in the code is this 2i vs i distance determined -- i.e. have we checked that the code matches this new doc_string?

Reply:
Woops, I meant to change them both to i - seems i'm finding this docstring tricky to get right, lol.

Reply:
The code has always been 'correct', it's just the docstring that was mistakenly a factor of 2 out because I was thinking of the neighborhood diameters rather than radius.